mantle/platform/util.go

```diff
@@ -157,6 +157,10 @@ func WaitForMachineReboot(m Machine, j *Journal, timeout time.Duration, oldBootI
 		} else if strings.Contains(err.Error(), "handshake failed: EOF") {
 			// This can also happen if we're killed during handshake
 			c <- nil
+		} else if strings.Contains(err.Error(), "dial tcp") {
+			// Catch "dial tcp xx.xx.xx.xx:xx: connect: connection refused"
+			// error which occcurs when rebotting on online platforms.
```

Comment:
suggested wording update and spelling fix:

```suggestion
			// error which can occur when rebooting on online platforms.
```